assets/js/googlesitekit/widgets/datastore/widgets.js

```diff
@@ -20,12 +20,14 @@
  * External dependencies
  */
 import invariant from 'invariant';
+import sortBy from 'lodash';
```

Comment:
Shouldn't this be 
```suggestion
import { sortBy } from 'lodash';
```
?

Reply:
I'm actually a bit surprised this didn't cause tests to fail 😢